apps/prairielearn/elements/pl-order-blocks/dag_checker.py

```diff
@@ -149,7 +149,7 @@ def add_edges_for_groups(
 
 
 def grade_dag(
-    submission: list[str],
```

Comment:
I checked some upstream callers, it can be `list[str | None]`.

Reply:
@SethPoulsen does this seem expected too? We could use some documentation about what `submission` actually is.

Reply:
Ya, it ends up have `None` in the list if the student selects a disctractor block (a block that is always wrong).